src/components/HamburgerMenu/HamburgerMenu.js

```diff
@@ -40,43 +40,42 @@ const HamburgerLine = styled.span`
   ${hamburgerLineMixin('50%', 'background-color 0.3s ease-in-out')};
 
   transform: translateY(-50%);
-  background-color: ${({ isClicked }) =>
-    isClicked ? 'transparent' : colors.beta};
+  background-color: ${({ isNavActive }) =>
+    isNavActive ? 'transparent' : colors.beta};
 
   &::before {
     ${hamburgerLineMixin(`${rem(-10)}`, 'transform 0.2s ease-in-out')};
 
     content: '';
-    transform: ${({ isClicked }) =>
-      isClicked && `translateY(${rem(10)}) rotate(45deg)`};
+    transform: ${({ isNavActive }) =>
+      isNavActive && `translateY(${rem(10)}) rotate(45deg)`};
   }
 
   &::after {
     ${hamburgerLineMixin(`${rem(10)}`, 'transform 0.2s ease-in-out')};
 
     content: '';
-    transform: ${({ isClicked }) =>
-      isClicked && `translateY(${rem(-10)}) rotate(-45deg)`};
+    transform: ${({ isNavActive }) =>
+      isNavActive && `translateY(${rem(-10)}) rotate(-45deg)`};
   }
 `;
 
-const HamburgerMenu = ({ navActiveToggle }) => {
-  const [isClicked, setClicked] = useState(false);
-  const handleClick = () => {
-    setClicked(!isClicked);
-    navActiveToggle();
-  };
-  return (
-    <HamburgerButton onClick={handleClick}>
-      <HamburgerBox>
-        <HamburgerLine isClicked={isClicked} />
-      </HamburgerBox>
-    </HamburgerButton>
-  );
-};
+const HamburgerMenu = ({ isNavActive, navActiveToggle }) => (
+  <HamburgerButton onClick={navActiveToggle}>
+    <HamburgerBox>
+      <HamburgerLine isNavActive={isNavActive} />
+    </HamburgerBox>
+  </HamburgerButton>
+);
 
 HamburgerMenu.propTypes = {
-  navActiveToggle: PropTypes.func.isRequired,
+  isNavActive: PropTypes.bool,
+  navActiveToggle: PropTypes.func,
+};
+
+HamburgerMenu.defaultProps = {
+  isNavActive: false,
+  navActiveToggle: () => {},
```

Comment:
if you have lodash, you could use `noop` to be cleaner

Reply:
don't have lodash installed - so I think adding it only for this one thing is not a good idea